go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/palantir/conjure-go/v6
 
-go 1.21
+go 1.23
```

Comment:
Can you update this to `go 1.23.0`? Newest format allows for full specification of version

Reply:
done!